docs/sdk/api-reference.md

```diff
@@ -3345,10 +2615,10 @@ Creates Annotation proto from keyword arguments.
 **Returns:**
 - Annotation proto message
 
-### Search._get_input_proto()
+### Search._get_input_proto
```

Comment:
I think this is a internal method. WDYT @sanjaychelliah ?

Reply:
This too

Reply:
As this is an internal method i have removed this from the doc